src/structures/command/Command.js

```diff
@@ -126,11 +126,13 @@ module.exports = class Command {
     return this.parentCommand ? `${this.parentCommand.fullName} ${this.name}` : this.name
   }
 
-  usage (t, prefix, noUsage = true) {
+  usage (t, prefix, noUsage = true, onlyCommand = false) {
     const usagePath = `${this.tPath}.commandUsage`
     const usage = noUsage ? t(`commands:${usagePath}`) : t([`commands:${usagePath}`, ''])
-    if (usage !== usagePath) {
-      return `**${t('commons:usage')}:** \`${prefix}${this.fullName} ${usage}\``
+    if (usage !== usagePath && !onlyCommand) {
```

Comment:
Why do you repeat `usage !== usagePath`? Use if inside another if instead, example:
```js
if (usage !== usagePath) {
  if (onlyCommand) {
    // ...
  } else {
    // ...
  }
}
```

Reply:
You realize I... Removed that line, right? That's why it's red